docs/rfcs/700-config-features-and-defaultpairs-via-connection-string.md

```diff
@@ -113,45 +113,31 @@ That means pair of operations with the same name `server_side_encryption` or `st
 
 ### Implementation
 
- Default pairs and features are defined in service.
+Default pairs and features are defined in service.
 
-**Add pairs for features and default paris**
+**Add `defaultable` in namespace**
 
-Default pairs and features can be added into `Optioanl` pairs of `New` for service, like:
+Add a new field `defaultable` in namespace to specify the defaultable pair key list, like:
 
 ```toml
-[namespace.storage.new]
-required = ["name"]
-optional = ["storage_features", "default_storage_pairs", "default_storage_class", "enable_virtual_dir"]
+[namespace.storage]
+features = ["virtual_dir"]
+implement = ["direr", "multiparter"]
+defaultable = ["excepted_bucket_owner"]
 ```
 
-When parsing pairs from toml, we should:
+Pair keys listed in `defaultable` will be treated as allowing users to set default values. When parsing toml files, we should:
 
-- Check whether the default pair belongs to global or system pair and feature belongs to the supported ones.
-- Add pairs for the optional pairs of `New` with `default_` and `enable_` prefix.
-- Mark them as defaultable, the same with global and system paris corresponding to the default ones. 
+- Check whether the key in `defaultable` belongs to the pair list of operations in the namespace.
+- Generate default pairs according to `defaultable` and `features` fields.
 
 **Parse features in `parsePair*New`**
 
-We can handle the optional defaultable pairs with prefix `enable_`, assign value to the corresponding field of `*Features` in `ParsePair*New()`.
+We can handle the defaultable pairs for features with prefix `enable_`, assign value to the corresponding field of `*Features` in `ParsePair*New()`.

```

Comment:
features will have `WithEnableVirtualDir`?

Reply:
Maybe `WithEnableVirtualDir()` is not necessary？I intended to add default paris for features to reduce the usage of `Feature Pairs Registry` for connection string, so `WithEnableVirtualDir()` will be generated if no additional controls.

Reply:
What does `add default paris for features` mean?

Reply:
Like pairs for `defaultable pairs`. Names of these pairs are prefixed with `enable_`. Then the keys `enable_xxx` will be added into `pairMap` and can be parsed when using connection string.

Reply:
OK, mostly LGTM.